### ISSUE TITLE
Can't Save Game Yomawari Midnight Shadows

### ISSUE THREAD

**Issue by Vastelordd** [Vita3K/Vita3K-Android#238]
![Screenshot_2023-02-18-18-39-14-661_org vita3k emulator](https://user-images.githubusercontent.com/125793436/219938492-ae867b5b-48e7-42be-b615-0d19685f8f31.jpg)

**Comment by KevinsHope** [comment]
Well...I got the same error...I think it is the emulation...not the game or the system it is running on.  Sucks too, because it looks like a great game. Hopefully they will fix the save issue, as they are aware that there are game breaking glitches with the emulation.

**Comment by DevDongoMakinRusak** [comment]
Yeah, all the same, tested it on v7 & 11. It's listed as playable, though.
